src/main/java/uk/ac/cam/cl/dtg/segue/api/InfoFacade.java

```diff
@@ -309,4 +313,21 @@ public Response pingETLServer(@Context final HttpServletRequest request) {
 
     }
 
+    @GET
+    @Path("/ping/{streams_app}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getUserStatsAppStatus(@Context final Request request, @PathParam("streams_app") final String appName) {
+
+        switch (appName) {
+            case "site_statistics":
+                return Response.ok(ImmutableMap.of("running", SiteStatisticsStreamsApplication.getAppStatus().get("running"))).build();
+
+            case "user_statistics":
+                return Response.ok(ImmutableMap.of("running", UserStatisticsStreamsApplication.getAppStatus().get("running"))).build();
+
+            default:
+                return Response.ok(ImmutableMap.of("running", false)).build();
```

Comment:
Would it not be more useful to return an exception if the streams app is not a valid appName?
We might think that the "site_statistics" app is down when it was actually just a typo/capital letter in the path param.

Reply:
Will change this now